### PR TITLE
ROCANA-3431: Create a crunch release script

### PR DESCRIPTION
**Pull request by joey**
- Add distributionManagement so that deploy target works
- Add HDP repositories to archetype pom file for tests to pass under
    certain mavent targets.
- Updated SCM connection to point to Rocana github.
- Added release script

### CODE REVIEW

**Comment by joey** [comment]
Tested with `CRUNCH_RELEASE_VERSION=0.13.0-hdp2.3.0-rocana1.3.0-rc4 CRUNCH_DEVELOPMENT_VERSION=0.13.0-hdp2.3.0-rocana1.3.0-SNAPSHOT ./rocana-release.s`. I ended up deploying, at least partially, 4 RCs during testing. Good thing integers are infinite!

File: pom.xml [hunk 1]
```diff
@@ -102,10 +102,10 @@ under the License.
...
 
   <scm>
-    <url>https://git-wip-us.apache.org/repos/asf?p=crunch.git</url>
-    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</connection>
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</developerConnection>
-    <tag>apache-crunch-0.13.0</tag>
+    <url>https://github.com/scalingdata/crunch</url>
+    <connection>scm:git:git@github.com:scalingdata/crunch.git</connection>
+    <developerConnection>scm:git:git@github.com:scalingdata/crunch.git</developerConnection>
+    <tag>0.13.0-hdp2.3.0-rocana</tag>
```
Comment (bradcupit) [inline pom.xml:108]: This seems different from the naming convention we used in Impala. Our Impala branch is named `cdh5.4.5-rocana1.3.0`
Should I change the name we used in Impala?

Comment (bradcupit) [inline pom.xml:108]: Oh wait, just re-read your above comment:

> Tested with CRUNCH_RELEASE_VERSION=`0.13.0-hdp2.3.0-rocana1.3.0-rc4`

I see, maven's `<tag>` is a branch name.
Nevermind my comment above, carry on. Nothing to see here.

**Comment by bradcupit** [comment]
+1

**Pull request merged**